cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+
+# OpenID Connect claim handlers
+OAUTH_OIDC_ID_TOKEN_HANDLERS = (
+    'oauth2_provider.oidc.handlers.BasicIDTokenHandler',
+    'oauth2_provider.oidc.handlers.ProfileHandler',
+    'oauth2_provider.oidc.handlers.EmailHandler',
+    # Includes the "administrator" claim in the token. This currently resides in lms/djangoapps!
+    'oauth2_handler.IDTokenHandler'
```

Comment:
A little more on what I've tried here. I've taken a debugger down into the depths of [edx/edx-oauth2-provider](https://github.com/edx/edx-oauth2-provider), looking specifically at how scopes are pulled from the handlers registered in `OAUTH_OIDC_ID_TOKEN_HANDLERS` ([oauth2_provider/oidc/collect.py#L50](https://github.com/edx/edx-oauth2-provider/blob/master/oauth2_provider/oidc/collect.py#L50)).

This exercise reveals that the `IDTokenHandler` is registered and is ["visited"](https://github.com/edx/edx-oauth2-provider/blob/master/oauth2_provider/oidc/collect.py#L96) during scope collection, but that the "permissions" scope is missing from the set of [candidate scopes](https://github.com/edx/edx-oauth2-provider/blob/master/oauth2_provider/oidc/collect.py#L48-L49).